migration/v0.32/README.md

```diff
@@ -35,6 +35,40 @@ As with all scripts in this repository `CK8S_CONFIG_PATH` is expected to be set.
 
 ## Prerequisites
 
+- If `.objectStorage.sync.enabled: true` and `.thanos.objectStorage.type: swift` or `.harbor.persistence.type: swift` then the rclone jobs will automatically use swift for Thanos and/or Harbor source and destination buckets. You will need to create the application credentials for swift and add them into the `secrets.yaml`:
```

Comment:
Doesn't `.objectStorage.sync.syncDefaultBuckets:` also have to be true for this or am i misunderstanding thanos and harbor.

Reply:
yes, that also should be enabled